packages/p2p/src/components/orders/chat/chat-message-receipt.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@deriv/components';
+import ChatMessage from 'Utils/chat-message';
+
+const ChatMessageReceipt = ({ chat_channel, message, sendbird_user_id }) => {
```

Comment:
Also here, how about `user_sendbird_id`?

Reply:
Same here, the `id` is referring to a `sendbird_user` not a `sendbird`.